frontend/client/components/Proposal/CampaignBlock/index.tsx

```diff
@@ -66,10 +66,10 @@ export class ProposalCampaignBlock extends React.Component<Props, State> {
 
       // Get bounty from RFP. If it exceeds proposal target, show bounty as full amount
       let bounty;
-      if (proposal.rfp && proposal.rfp.bounty) {
-        bounty = proposal.rfp.bounty.gt(proposal.target)
+      if (proposal.contributionBounty && proposal.contributionBounty.gtn(0)) {
```

Comment:
Can we give contributionBounty the same treatment as bounty, just leave it out if it's zero?

Reply:
Could be done, would make our frontend api BNification and Zatting functions a bit more complex.

Reply:
I also like always being able to run math on the field w/o checking if it exists.

Reply:
I don't think we use contributionBounty anywhere else, so I'm ok leaving this for now and getting the merge in. We can circle back if needed